instruments/thorlabs/thorlabsapt.py

```diff
@@ -1407,7 +1407,7 @@ def home_parameters(self, values):
             # replace values that are `None`
             if None in values:
                 set_params = self.home_parameters
-                values = [x if x else y for x, y in zip(values, set_params)]
+                values = [x if x is not None else y for x, y in zip(values, set_params)]
```

Comment:
Here we need to check in fact `if x is not None` and not just `if x`, otherwise an offset of zero cannot be set...

Reply:
:) thanks!